modules/nw-metalLB-basic-upgrade-operator.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+
+[id="upgrading-metallb-operator-to-minor-version_{context}"]
+= Upgrading the MetalLB Operator
+
+Manually upgrade the MetalLB Operator to the next minor version by using the {product-title} web console.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Access the web console and navigate to *Operators* -> *Installed Operators*.
+
+. Click *MetalLB Operator* to open the *Operator details* page.
+
+. Update the *Update approval* strategy.
++
+[NOTE]
+====
+When updating from version less than 4.11 set the *Update approval* strategy to manual. The automatic update mechanism is not working as expected.
+====
+
+.. Click *Subscription* to reveal the *Subscription details*. 
```

Comment:
Can we defer to https://docs.openshift.com/container-platform/4.10/operators/admin/olm-deleting-operators-from-cluster.html#olm-deleting-operators-from-a-cluster asking the user to choose their method of choice? I'd say not everyone wants to use the web ui